integration/util/scenario/chain.js

```diff
@@ -181,6 +181,12 @@ class Chain {
     });
   }
 
+  async tokenBalance(token, chainAccount) {
+    console.log(["gghy88", token.toChainAsset(), chainAccount]);
```

Comment:
?

Reply:
FWIW, there are so many logs in integration tests that I use random strings so I can easily "find" a message in the clutter 🤦